src/pages/en/legal/modmail-privacy-policy.mdx

```diff
@@ -87,21 +85,15 @@ your personal data, under certain conditions.
 
 If you would like to exercise those rights, contact us at [privacy@quiltmc.org](mailto:privacy@quiltmc.org)
 or by sending a message to the ModMail+ Discord Bot.
-We may ask you to verify your identity before processing your request, using a service such as [Aperture](https://aperture.starchild.systems).
+We may ask you to verify your identity before processing your request.
 We will respond to your request within 30 days as required by law.
 
-# Analytics
-
-We use [Cloudflare Web Analytics](https://www.cloudflare.com/en-gb/web-analytics/) to
-collect anonymous usage statistics of the web interface.
-Cloudflare does not collect or process personal data for Us.
```

Comment:
Loading the website, it appears a request is still sent to `https://quiltmc.org/cdn-cgi/rum`. This indicates Cloudflare Web Analytics are still in use.

Reply:
I believe this has been fixed, at least I do not see said request when loading the site anymore.